src/components/spreadsheet/mixins/copyPaste.js

```diff
@@ -446,7 +446,7 @@ export const copyPaste = {
 
       // Position bottom/top of rectangle if rowStart >= rowEnd
       if (this.selectedCoordCells.rowStart >= this.selectedCoordCells.rowEnd) {
-        this.rectangleSelectedCell.style.setProperty('--rectangleTop', 'auto')
+        this.rectangleSelectedCell.style.setProperty('--rectangleTop', 0)
```

Comment:
@ninetteadhikari I tried this too but i think it also breaks when you hightlight from bottom to top.